docker/ravendb-ubuntu1604/Dockerfile

```diff
@@ -9,14 +8,17 @@ ENV RAVEN_ARGS='' RAVEN_SETTINGS='' RAVEN_Setup_Mode='Initial' RAVEN_DataDir='Ra
 
 EXPOSE 8080 38888 161
 
+ADD https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt /usr/share/ca-certificates/lets-encrypt-x3-cross-signed.pem
```

Comment:
Don't we nee this one ? 

http://apps.identrust.com/roots/dstrootcax3.p7c
--

Reply:
The certificate is already there. I think we lacked intermediate one locally.
See http://issues.hibernatingrhinos.com/issue/RavenDB-11316#comment=67-29057

Reply:
Did you check the remote calls there?

Reply:
Was busy fixing release, will do that today.